subprojects/packagefiles/yajl/meson.build

```diff
@@ -1,4 +1,9 @@
 project('YetAnotherJSONParser', 'c', version: '2.1.0')
+
+if get_option('buildtype') != 'static'
```

Comment:
Do you mean `default_library`?

Reply:
Yep. Turns out there are 12 more places in here with the same mistake.

Reply:
Fixed.